pages/sdk.js

```diff
@@ -79,6 +78,39 @@ class SDK extends React.Component<OSProps> {
               </Type.p>
             </ContentSection.Pane>
           </ContentSection>
+          <ContentSection>
```

Comment:
Can you add a title or some other separation between the previous pane and this one? It looks a little choppy in the UI at the moment.